winml/lib/Api.Ort/OnnxruntimeEngineBuilder.cpp

```diff
@@ -42,6 +42,11 @@ STDMETHODIMP OnnxruntimeEngineBuilder::CreateEngine(_winml::IEngine** out) {
                             ort_api);
   }
 
+  if (intra_op_num_threads_override_.has_value()) {
+    RETURN_HR_IF_NOT_OK_MSG(ort_api->SetInterOpNumThreads(session_options.get(), intra_op_num_threads_override_.value()),
```

Comment:
Set **Inter** OpNumThreads([..]] **intra_** op_num_threads_override_.value()

Should it be inter or intra? Before your change WinML was calling ort_api->SetIntraOpNumThreads

Reply:
Ahh yeah should be **intra_**, my mistake.